src/app/config/api-config.ts

```diff
@@ -39,4 +40,5 @@ export const API_CONFIG: ApiConfig = {
   embeddedProjectRoute: 'embed',
   wizardRoute: 'wizard',
   uploadFileRoute: 'file',
+  projectLikes: 'project/like/'
```

Comment:
I think you meant:
```suggestion
  projectLikes: 'project/like'
```
so without the '/'. The HttpBaseService will add the slash for you, otherwise it results in a double slash I think...?